wouso/resources/templates/cpanel/races/all.html

```diff
@@ -22,45 +22,191 @@
-                  <span class="glyphicon glyphicon-edit"></span>
-              </a>
-              <a href="{% url system_message_group g.id %}"
-                 title="Send a message to this group">{% trans 'Message' %}
-              </a>
-          </li>
-        {% endfor %}
-      </ul>
-    </li>
-</ul>
+    
+    {% endfor %}
+
+    <!-- Display orphan race -->
+    <tr>
+        <td class="ctr-number">{{races.count}}</td>
+        <td>Orphans</td>
+        <td></td>
+        <td>{{orphan_groups.count}}</td>
+        <td></td>
+        <td>
+            <div class="row">
+                <div class="col-md-1"></div>
+                <div class="col-md-5 right">
+                    <a class="btn btn-success btn-sm disabled">
+                        <span class="glyphicon glyphicon-edit"></span> {% trans 'Edit '%}
+                    </a>
+                </div>
+                <div class="col-md-5">
+                    {% if orphan_groups.count %}
+                        <a class="toggle_groups btn btn-warning btn-sm btn-block" data-show="1" data-race="race-orphans">
+                            <span class="glyphicon glyphicon-eye-open"></span> {% trans 'Show groups' %}
+                        </a>
+                    {% else %}
+                        <a class="btn btn-warning btn-sm disabled btn-block">
+                            <span class="glyphicon glyphicon-eye-open"></span> {% trans 'Show groups' %}
+                        </a>
+                    {% endif %}
+                </div>
+                <div class="col-md-1"></div>
+            </div>
+        </td>
+    </tr>
+    <!-- Display orphan groups -->
+    {% for g in orphan_groups %}
+        <tr class="group group-row race-orphans">
+            <td>{{races.count}}.{{forloop.counter0}}</td>
+            <td>{{g}}</td>
+            <td>{{g.title}}</td>
+            <td>n/a</td>
+            <td></td>
+            <td>
+                <div class="row">
+                    <div class="col-md-1"></div>
+                    <div class="col-md-5 right">
+                        <a href="{% url admin:user_playergroup_change g.id %}" class="btn btn-success btn-sm">
+                            <span class="glyphicon glyphicon-edit"></span> {% trans 'Edit '%}
+                        </a>
+                    </div>
+                    <div class="col-md-5">
+                        <a href="{% url system_message_group g.id %}" class="btn btn-info btn-sm btn-block"> 
+                            <span class="glyphicon glyphicon-envelope"></span> {% trans 'Send message' %}
+                        </a>
+                    </div>
+                    <div class="col-md-1"></div>
+                </div>
+            </td>
+        </tr>
+    {% endfor %}
+</table>
+
+<link rel="stylesheet" type="text/css" href="{{ basepath }}/static/css/cpanel-style.css" />
+<script type="text/javascript" src="{{ basepath }}/static/js/jquery-ui-1.8.16.custom.min.js"></script>
```

Comment:
Because this file extends the cpanel/index.html template, it also contains any imported stylesheets and scripts. JQuery and cpanel-style.css are already loaded there:
https://github.com/catalin-oancea/wouso/blob/groups_ui/wouso/resources/templates/cpanel/index.html#L20
https://github.com/catalin-oancea/wouso/blob/groups_ui/wouso/resources/templates/cpanel/index.html#L16